kairon/api/app/routers/bot/customer_orders.py

```
@@ -152,12 +160,13 @@ async def list_orders_for_customer(
 
 @router.post("/orders/filter", response_model=Response)
 async def filter_orders(
+        bot: str,
         request_data: FilterOrdersRequest,
-        current_user: User = Security(Authentication.get_current_user_and_bot, scopes=TESTER_ACCESS),
+        current_user: User = Security(Authentication.validate_store_page_token, scopes=TESTER_ACCESS),
```

Ruff (0.16.1)

[warning] Do not perform function call `Security` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)